### ISSUE TITLE
[Bug] plq_Ridge_Classifier raises Exception for 'monotonic' constraint

### ISSUE THREAD

**Issue by BenViolet6** [softmin/ReHLine-python#38]
## Bug Description
I am attempting to train a plq_Ridge_Classifier with an SVM loss and a monotonicity constraint. However, when passing {'name': 'monotonic'} to the constraint parameter, the fit method fails with an exception stating that the constraint is not supported by ReHLine.

## Expected Behavior
The plq_Ridge_classifier may natively support the {'name': 'monotonic'} constraint configuration. I expected the library to automatically construct the necessary constraint matrix (A) and vector (b) to enforce monotonicity on the coefficients, allowing the fit method to complete successfully without requiring the user to manually derive custom linear constraints.

## Actual Behavior
The code raises an Exception during the fit process: Exception: Sorry, ReHLine currently does not support this constraint, but you can add it by manually setting A and b via {'name': 'custom', 'A': A, 'b': b}

## Steps to Reproduce

1. Generate synthetic classification data (using make_classification).
2. Initialize plq_Ridge_Classifier with loss={'name': 'svm'}.
3. Set the constraint parameter to [{'name': 'monotonic'}].
4. Run mclf.fit(X, y).

## Minimal Code Example

```python
# Minimal reproducible example
import numpy as np
from rehline import ReHLine

# Minimal reproducible example
import numpy as np
from sklearn.datasets import make_classification
from sklearn.preprocessing import StandardScaler
from rehline import plq_Ridge_Classifier

# 1. Simulate data
n, d = 10000, 5
X, y = make_classification(n_samples=n, n_features=d, n_redundant=0, random_state=42)
y = 2 * y - 1 
scaler = StandardScaler()
X = scaler.fit_transform(X)

# 2. Solve SVM with Monotonicity Constraint
mclf = plq_Ridge_Classifier(
    loss={'name': 'svm'},
    constraint=[{'name': 'monotonic'}],
    C=1.0,
    max_iter=10000
)

# This line triggers the exception
mclf.fit(X=X, y=y)
```

## Environment

- OS: Linux / 6.6.113+
- Python version: 3.12.12
- ReHLine version: 0.1.1
- NumPy version: 2.0.2
- SciPy version: 1.16.3
- scikit-learn version: 1.6.1
